redis_docker.conf

```diff
@@ -0,0 +1,9 @@
+# TODO(#18260): This file is used within the Docker setup for local development.
+# Modify the `redis.conf` file when we permanently switch to Docker.
```

Comment:
This issue doesn't mention redis_docker.conf or redis.conf

Reply:
Thanks @U8NWXD, I have updated the issue desc. to remove the extra config files too that are required for the Docker setup during the transition period